translations/README.md

```diff
@@ -14,44 +14,42 @@ Refer to GitHub docs for more information on [proposing changes via pull request
 
 A good way to assure the quality of the translations is to build the docs with the up-to-date message catalog (.po file). If you are willing to down that way, follow the steps below.
 
-**Note:** All instructions below assume you are inside `translations/` directory.
+**Note:** All instructions below assume you are running from the project's root directory.
 
-1. Install dependencies (hint: add `--user` flag for a per-user installation, without requiring admin privileges):
+1. Install `tox` (hint: add `--user` flag for a per-user installation, without requiring admin privileges):
 ```
-pip install -r ../requirements.txt
+pip install tox
 ```
 
-2. Copy your PO file to `po/` (replace `<lang>` accordingly):
+2. Copy your .po file to `translations/po/` (replace `<lang>` accordingly):
 ```
-cp path/to/your/translated/file.po po/<lang>.po
+cp path/to/your/translated/file.po translations/po/<lang>.po
 ```
 
-3. If you are adding new translation for a language, add your language code to `LINGUAS` in alphabetical order; otherwise, skip this step.
+3. If you are adding new translation for a language, add your language code to the `LINGUAS` file in alphabetical order; otherwise, skip this step.
 
-4. Split your PO file into individual documents for Sphinx to process:
+4. Build HTML docs in your language by running:
 ```
-make split-po
+tox
 ```
 
-5. Build HTML docs in your language (replace `<lang>` accordingly):
-```
-make -C .. -e SPHINXOPTS="-D language='<lang>'" html
-```
-
-4. Check `../build/html/` for the translated docs in HTML format.
+5. Check `build/html/` for the translated docs in HTML format.
 
 ## Building translation template
 
-This step is **not** necessary for translating PyGObject-Tutorial, as [Damned Lies](https://l10n.gnome.org/) automatically updates the translation template (.pot file) and the message catalog (po) files. Therefore, it should be all up-to-date already.
-
-1. Install dependencies (same instruction as step 1 from previous section)
+This is **not** necessary for translating PyGObject-Tutorial, as [Damned Lies](https://l10n.gnome.org/) automatically updates the translation template (.pot file) and the message catalogs (.po files). Therefore, it should be all up-to-date already.
 
-2. Run:
+1. To build the .pot file, run:
 ```
-make PythonGTK3Tutorial.pot
+make -C translations PythonGTK3Tutorial.pot
 ```
 
-3. Check the translation template as `PythonGTK3Tutorial.pot`
+2. If the execution is done successfully, then `translations/PythonGTK3Tutorial.pot` should be available.
+
+3. If step 1 failed, run the following command before retrying step 1:
```

Comment:
Generation of the .pot file only requires gettext to be installed, no python dependencies.

Reply:
Thanks. Step 3 removed as it is unnecessary.